frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -1004,6 +1004,9 @@ function ReaderRolling:_gotoPos(new_pos, do_dim_area)
     else
         self.view.dim_area:clear()
     end
+    if self.current_pos and not UIManager.currently_scrolling then
+    self.ui:handleEvent(Event:new("PageChangeAnimation", new_pos > self.current_pos))
```

Comment:
indentation.

Will this do a horizontal (left/right) swipe animation, or vertical ? Feels like when in scroll mode, it should be vertical - but I don't seem to see support for vertical in here or in the base PR.

Reply:
horizontal.

This only happens when changing a page (via a left/right tap or swipe) so it makes sense to go L/R.
When panning (U/D) this is disabled as the full screen animation interferes with the pan refresh rate.

> but I don't seem to see support for vertical in here or in the base PR.

I did not add it as it is not need, but very easy to add in later if wanted.